src/containers/Editor/components.tsx

```diff
@@ -170,7 +179,17 @@ const EditorContainer: React.FC<EditorContainerProps> = ({
       if (index < 0 || index >= project.accounts.length) {
         return
       }
-      return project.accounts[index].draftCode
+
+      // combine the account's contracts as one string
+      // and return for cadence server check
+      var combinedCode = '';
+      for (let i = 0; i < project.contracts.length; i++) {
+        if(project.contracts[i].index == index) {
+          combinedCode += project.contracts[i].script;
+        }
+      }
+
+      return combinedCode;
```

Comment:
Why we are doing this? 🤔

Reply:
code is passed to Cadence server to check for available declarations/contracts deployed at an account. this is to fix the earlier issue noted here: https://user-images.githubusercontent.com/81572453/140640705-79b27723-0a13-444b-b7cb-5bb78b3ae1b0.png

Reply:
previous only the account's code (1 contract) was passed for declaration. since we now have multiple contracts (hence multiple declarations) we need to pass all contracts of an account, combined. this is just for frontend check.